templates/product_management/create_product_profile.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+    <div class="content-i">
+        <div class="content-box">
+            <div class="element-wrapper">
+                <div class="element-box">
+                    <form method="post">
+                        {% csrf_token %}
+                        <div class="steps-w">
+                            <div class="step-triggers">
+                                <a class="step-trigger active" href="#stepContent1">
+                                    Create Product
+                                </a>
+                            </div>
+                            <div class="step-contents">
+                                <div class="step-content active" id="stepContent1">
+                                    <div class="row">
+                                        <div class="col-sm-6">
+                                            <div class="form-group">
+                                                <label for=""> Name </label>
+                                                <input class="form-control" name="title"
```

Comment:
You might want to update html files as well to shows description field.

Reply:
Already been added in last commit.

`                                    <div class="row">

                                        <div class="col-sm-6">
                                            <div class="form-group">
                                                <label> Description </label>
                                                <textarea class="form-control" rows="5" name="description"
                                                          placeholder="Anything you would like to add."> {{ product.description }} </textarea>
                                            </div>
                                        </div>
                                    </div>`